stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -4,95 +4,114 @@ This package provides the functionality required to access and manipulate data s
+    username: "root",
+    password: "root",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
 };
 ```
 The full list of endpoint properties can be found in the `sql:PoolOptions` type.
 
 ### Creating tables
 
-Following is an example of creating a table with two fields. One field is of type int and the other of varchar. The CREATE statement is executed via the `update` operation of the endpoint.
+Following is an example of creating a table with two columns. One field is of type int and the other of varchar. The CREATE statement is executed via the `update` operation of the endpoint.
 
 ```ballerina
-//Create ‘Students’ table with fields ‘StudentID’ and ‘LastName’. 
-int returnValue = check testDB->update("CREATE TABLE IF NOT EXISTS Students(StudentID int, LastName varchar(255))");
-
-//Terminate the connection pool.
-testDB.stop();
+//Create ‘Students’ table with fields ‘StudentID’ and ‘LastName’.
+var ret = testDB->update("CREATE TABLE IF NOT EXISTS Students(StudentID int, LastName varchar(255))");
+match ret {
+    int retInt => io:println("Students table create status in DB: " + retInt);
+    error err => io:println("Students table create failed: " + err.message);
+}
 ```
 
-### Selecting data
+### Inserting data
 
-Following is an example of selecting data. First, a type is created to represent the returned result set. Then the SELECT query is execute via the `select` operation of the endpoint by passing that result set type. Once the query is executed, each data record can be retrieved by looping the result set.
+Following are three examples of data insertion by executing an INSERT statement using the `update` operation of the endpoint. 
 
-```ballerina
-//Define a type to represent the results set.
-type ResultCustomers {
-    string FIRSTNAME,
-};
-
-//Retrieve the FirstName from the Customers table by providing the registrationID.
-table dt = check testDB->select("SELECT  FirstName from Customers where registrationID = 1", ResultCustomers);
-string firstName;
+In the first example, query parameter values are directly passed into the query statement of the `update` operation:
 
-//Iterate the results-set and retrieve the first name.
-while (dt.hasNext()) {
-    ResultCustomers rs = check <ResultCustomers>dt.getNext();
-    firstName = rs.FIRSTNAME;
+```ballerina
+var ret1 = testDB->update("INSERT INTO Students(StudentID, LastName) values (1, 'john')");
+match ret1 {
+    int retInt => io:println("Inserted row count to Students table: " + retInt);
+    error err => io:println("Insert to Students table failed: " + err.message);
 }
-testDB.stop();
 ```
 
-### Inserting data
+In the following second example, parameter values which are in local variables are directly passed as parameters to the `update` operation. This direct parameter passing can be done for any primitive ballerina type like string, int, float, boolean. The sql type of the parameter is derived from the type of the ballerina variable passed in. 
```

Comment:
Change "following second" to "next"
Change "parameters values which are" to "parameter values that are"
Change "ballerina" to "Ballerina" (two instances in this paragraph)
Change "float, boolean" to "float, or boolean"